src/lib.rs

```diff
@@ -61,6 +75,12 @@ pub struct SimpleLogger {
     /// This field is only available if the `color` feature is enabled.
     #[cfg(feature = "colored")]
     colors: bool,
+
+    /// Whether to use thead identifiers or not.
+    ///
+    /// This field is only available if the `thread_ids` feature is enabled.
+    #[cfg(feature = "thread_ids")]
+    threadids: bool,
```

Comment:
This attribute should be `thread_ids` to match the feature name.

Reply:
will do, but thats part of #39